tracer/src/Datadog.Trace/ClrProfiler/CallTarget/CallTargetInvoker.cs

```diff
@@ -29,6 +29,7 @@ public static class CallTargetInvoker
         {
             if (IntegrationOptions<TIntegration, TTarget>.IsIntegrationEnabled)
             {
+                IntegrationOptions<TIntegration, TTarget>.RecordTelemetry();
```

Comment:
We have to record that ourselves using `Telemetry.IntegrationRunning(IntegrationId)`

Reply:
Instead of doing it manually, shouldn't we do the same on OnMethodEnd in addition to OnMethodBegin? Especially now that we don't go through the Interlocked at each call

Reply:
Oh, I see what you mean, I thought you meant where we didn't have the direct calltarget instrumentation at all! Yeah, that makes sense 👍

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2241/commits/63e88a2612271568f8fbd373830acc9bf52d841d